controllers/toolchainconfig/toolchainconfig_controller.go

```diff
@@ -3,19 +3,27 @@ package toolchainconfig
 import (
 	"context"
 	"fmt"
+	"os"
 	"time"
 
 	"github.com/go-logr/logr"
+	v1 "github.com/openshift/api/template/v1"
```

Comment:
can we have `templatev1` as an alias? (like we have `corev1`, `metav1`, etc.)

Reply:
done in 853e82cf304e083fe7a50390404297f8c81a1546